pkg/cointicker/cointicker.go

```diff
@@ -32,28 +32,31 @@ func NewCoinTicker(client *cmcap.CoinMarketClient, cache *storage.Cache) CoinTic
 
 func (c CoinTicker) GetFormattedPrice(t time.Time) (string, []error) {
 
-	btc, eth, ratio, errors := c.generateResult()
+	btc, eth, xmr, ethRatio, xmrRatio, errors := c.generateResult()
 
 	if len(errors) != 0 {
 		return "", errors
 	}
 
 	lastEntry := c.Cache.GetLast()
-	c.Cache.AddEntry(btc, eth, float.Round(ratio, .5, 6), t.UTC())
+	c.Cache.AddEntry(btc, eth, xmr, float.Round(ethRatio, .5, 6), float.Round(xmrRatio, .5, 6), t.UTC())
 
-	var f func(interface{}) aurora.Value
-	if ratio > lastEntry.Ratio {
-		f = c.au.Green
+	decorateRatio(ethRatio, lastEntry.ETHRatio, c)
+
+	return fmt.Sprintf("%s BTC: %s, ETH: %s, XMR: %s \nB/E ratio %f, B/M ratio %f \n\n", t.Format(timeFormat), btc, eth, xmr, decorateRatio(ethRatio, lastEntry.ETHRatio, c)(ethRatio), decorateRatio(xmrRatio, lastEntry.XMRRatio, c)(xmrRatio)), nil
+}
+
+func decorateRatio(r, lr float64, c CoinTicker) func(interface{}) aurora.Value {
+	if r > lr {
+		return c.au.Green
 	} else {
-		f = c.au.Red
+		return c.au.Red
 	}
-
-	return fmt.Sprintf("%s BTC: %s, ETH: %s, ratio %f \n", t.Format(timeFormat), btc, eth, f(ratio)), nil
 }
 
-func (c CoinTicker) generateResult() (btc, eth string, ratio float64, errors []error) {
+func (c CoinTicker) generateResult() (btc, eth, mnr string, ethRatio, mnrRatio float64, errors []error) {
 
-	coins, errors := c.Client.GetCurrenciesQuotes([]string{cmcap.Bitcoin, cmcap.Ether}...)
+	coins, errors := c.Client.GetCurrenciesQuotes([]string{cmcap.Bitcoin, cmcap.Ether, cmcap.Monero}...)
```

Comment:
And that's the beauty of this API: to add a fetching for a new coin, you just need to add it in this method call :)